source/geometry/src/TG4GeometryManager.cxx

```diff
@@ -287,8 +288,7 @@ void TG4GeometryManager::ConstructG4Geometry()
   }
 
   // VMC application construct geometry
-  if (fUserGeometry == "RootToGeant4") {
-
+  if (fUserGeometry == "RootToGeant4" && !fUseExternalGeometryConstruction) {
```

Comment:
Multiple MC's are not supported with "RootToGeant4", so with this option
should be fUseExternalGeometryConstruction always "false" and there is no point to add 
this test; and as this is the only place where the added data member is used,
it looks like being redundant, including its setter